doc/site/content/docs/guides/rendering/lua-vbo-vao.md

```diff
@@ -41,8 +35,8 @@ Rectangles have 4 vertices, but since GPU draws with triangles, rectangle needs
 
 ![image]({{ site.baseurl }}/assets/guides/lua-vbo-vao-2.png)
```

Comment:
This failed to render

https://slashscreen.github.io/RecoilEngine/docs/guides/rendering/lua-vbo-vao/